runtime/kusama/src/weights/frame_election_provider_support.rs

```diff
@@ -48,20 +48,20 @@ impl<T: frame_system::Config> frame_election_provider_support::WeightInfo for We
 	/// The range of component `t` is `[500, 1000]`.
 	/// The range of component `d` is `[5, 16]`.
 	fn phragmen(v: u32, _t: u32, d: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 56_000
-			.saturating_add(Weight::from_ref_time(13_944_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 4_876_000
-			.saturating_add(Weight::from_ref_time(2_223_649_000 as u64).saturating_mul(d as u64))
+		Weight::from_ref_time(5_472_372_000 as u64)
+			// Standard Error: 138_792
+			.saturating_add(Weight::from_ref_time(5_519_912 as u64).saturating_mul(v as u64))
+			// Standard Error: 14_189_642
+			.saturating_add(Weight::from_ref_time(1_536_370_329 as u64).saturating_mul(d as u64))
 	}
 	/// The range of component `v` is `[1000, 2000]`.
 	/// The range of component `t` is `[500, 1000]`.
 	/// The range of component `d` is `[5, 16]`.
 	fn phragmms(v: u32, _t: u32, d: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 79_000
-			.saturating_add(Weight::from_ref_time(14_480_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 6_844_000
-			.saturating_add(Weight::from_ref_time(2_525_332_000 as u64).saturating_mul(d as u64))
+		Weight::from_ref_time(4_270_710_000 as u64)
```

Comment:
Is then being printed as 
![image](https://user-images.githubusercontent.com/10380170/197229594-7b36e397-a9a9-4e91-ae0b-c836653c048b.png)

This comes from a recent change in Substrate, since the weight calculating did not take into account that not all components can even start at 0.